### ISSUE TITLE
Management Model Improvements and Ideas

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/pyBRAT#17]
@bangen please post what your ideas and such are for improvements and refinements to the Management model here.

Discussion Points:

- I would like to see the management model expanded to break out 'restoration/conservation' zones into separate restoration and conservation zones, based on the `eDAMPcC` [attributes](http://brat.riverscapes.xyz/Documentation/10-WorkingWithBRATOutputs.html#attribute-field-descriptions). In other words, if beaver are there, and conflict is low, you want to keep encouraging.
- Also, we should look at 1 - `eDAMPcC` [attributes](http://brat.riverscapes.xyz/Documentation/10-WorkingWithBRATOutputs.html#attribute-field-descriptions) as an indication of additional capacity!
- Sara @bangen points out how land use is not really being used meaningfully. 
- Can we do a better job of providing an input of where 'incised (stage 1) channels' are?

**Comment by wally-mac** [comment]
Scott Shahverdian spent some time messing around with the BRAT management model (conceptually, not the code), and he thinks it makes sense, but now **h**e's ready for us to tell him what's wrong with it.****

He decided to really start from the beginning, in order to question some things that might prove to open a few doors that were previously closed. The biggest change is that he's basing everything on a new approach where we display both the management layer AND the conflict potential layer simultaneously, rather than attempt to incorporate them into a single layer. He thinks this makes our outputs more clear, allows for more nuance, and allows the user to incorporate local knowledge re: conflict potential that we may or may not have access to. Also, it actually eliminates a few categories, rather than creating new categories.

He's approach is outlined in a PowerPoint  Check it out, let him know if you think this approach is tractable, and/or if the logic is sound. He  tried to provide enough detail that it is intelligible without an in-person explanation, but you can be the judge of that. 
[BRAT_mgmt_new_categories.pptx](https://github.com/Riverscapes/pyBRAT/files/1993146/BRAT_mgmt_new_categories.pptx)

Here is a PNG of the logic as well: 
![mgmt_categories_new](https://user-images.githubusercontent.com/21991506/39893065-d5574c98-545f-11e8-8d5d-aba3c6e4d109.png)



Holler with questions,

**Comment by wally-mac** [comment]
I believe that @bangen has this new logic coded in PyBRAT and that for the upcoming TNC BRAT Workshop (May 24-25) that we'll be able to interrogate the outputs for the priority watersheds.  Is this the case?

**Comment by bangen** [comment]
@joewheaton @wally-mac I made some edits to Scott's slides.  They're attached below.  

Scott did an awesome job reworking the management model.  IMO, it's a huge improvement over the previous version and should be our starting point moving forward.  

From the runs for the 5 basins in the Sierra Nevada I found there are some gaps/missing categories for:

- Instances where existing capacity is greater than potential capacity.  For now we're designating those as 'Suitable Conditions: Maintain Habitat'.  We came up with the name on the fly, so I'm sure we can come up with a better name
- Instances where reach doesn't fall within any of the current designated categories.  Tend to be reaches where land use is moderate (i.e., agriculture between 20-50% and developed between 10-50%).   This instance is only seen in very small number of reaches (<0.75% for a few basins I checked).  For now we simply categorized these as 'Need Categorical Definition'

A few things to note:
- I made some edits to the slides, but left the 'old category' matrices that Scott put together untouched.  I'm pretty sure they reflect the original management model and not the one that Wally and I re-worked a few weeks back
- The code for this version of the management model is not in the BRAT toolbox yet.  Since there are currently some gaps and the prospect that we will make changes at the Truckee meeting it seemed premature to throw it into the toolbox.

I attached a kmz below for the Truckee

I was going to tag Scott here, but appears he isn't on Git?

[BRAT_mgmt_new_categories.pptx](https://github.com/Riverscapes/pyBRAT/files/2018479/BRAT_mgmt_new_categories.pptx)

[BRAT_ManagementModel_Truckee_051818_version..zip](https://github.com/Riverscapes/pyBRAT/files/2018491/BRAT_ManagementModel_Truckee_051818_version.zip)
